WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -399,7 +561,10 @@ private extension SupportTableViewController {
 
     struct Constants {
         static let appSupportURL = URL(string: "https://apps.wordpress.com/mobile-app-support/")
+
+        // TODO can be removed after FeatureFlag.wordPressSupportForum is removed
```

Comment:
I think we can remove `forumsURL` now, since this link hasn't been working for a long time, WDYT?

Reply:
@staskus sounds good and that's done now

Reply:
Thanks!